ovos_padatious/id_manager.py

```
@@ -49,8 +50,12 @@ def save(self, prefix):
             json.dump(self.ids, f)
 
     def load(self, prefix):
-        with open(prefix + '.ids', 'r') as f:
-            self.ids = json.load(f)
+        if os.path.isfile(prefix + '.ids'):
+            try:
+                with open(prefix + '.ids', 'r') as f:
+                    self.ids = json.load(f)
+            except json.JSONDecodeError as e:
+                raise ValueError(f"Invalid JSON in {prefix}.ids: {e}")
```

Ruff (0.8.0)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)